### ISSUE TITLE
External: Windows Exec is Ineffective - PS is inescapable

### ISSUE THREAD

**Issue by orcmid** [microsoft/vscode#130606]
Does this issue occur when all extensions are disabled?: Yes

aiso VS Code Version 1.59.0
VS Code Version: 1.60.0-insider (user setup)
Commit: 7c25c174726ab969db3f99058cdf24bcfae1cfbe
Date: 2021-08-11T05:15:24.598Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0

OS Version
OS: Windows_NT x64 10.0.19043

Steps to Reproduce:

In settings, change Explorer Kind to external
2. Leave the External: Windows Exec to the prefilled "C:\\Windows\\Sysem32\\cmd.exe" setting
3. close, reopen or whatever the change bar seems to stay there
4. Use Terminal | New Terminal

Expected: cmd opens.
Unexpected: PS and the commercial for PS ha;ppens.
Can switch to cmd (illustrated) but not what is desired.

![VSCode-1 60 0-2021-08-11-1200-insider--externalsettingl](https://user-images.githubusercontent.com/3372580/129090177-d52001aa-824d-49d6-ba3d-897937c70e0c.png)

**Comment by IllusionMH** [comment]
/question

> In settings, change Explorer Kind to external

You are changing wrong setting. Explorer Kind only controls what will be launched if you trigger context menu in Explorer view
![image](https://user-images.githubusercontent.com/1312662/129100987-0855b401-e019-4f31-bb35-9f1e836d0bcd.png)

---

As for PS - you need to configure `terminal.integrated.shell.windows`  and `terminal.integrated.shellArgs.windows` (which are still available in 1.59, but will be removed in next release) or new `terminal.integrated.profiles.windows` and `terminal.integrated.defaultProfile.windows` settings (however they might be affected by #130268 which is fixed only in Insiders).

**Comment by orcmid** [comment]
> 
> You are changing wrong setting. Explorer Kind only controls what will be launched if you trigger context menu in Explorer view
> As for PS - you need to configure:
> `terminal.integrated.shell.windows` and `terminal.integrated.shellArgs.windows` (which are still available in 1.59, but will be removed in next release)
> or
> new `terminal.integrated.profiles.windows` and `terminal.integrated.defaultProfile.windows` settings (however they might be affected by #130268 which is fixed only in Insiders).

@IllusionMH Thanks for this.  If I only had the triage-bot message I would have stopped supporting VS Code for beginner C++ GameDev courses (which already promote mingw/g++ :(

**Comment by IllusionMH** [comment]
I've trigger bot message (which closes issue) as I think that original issue can be resolved with proper configuration described in answer and there is no need to keep it open.

If it doesn't resolve the problem and there is definitely something wrong with editor itself you can provide more information to have issue reopened.

**Comment by orcmid** [comment]
@IllusionMH I understand what was done triggering the bot.  It assumes of course that the bot message makes sense to me.  I had no idea I was using the wrong feature.  So I would have to figure out some sort of search that got me an answer to "how can I get VSCode Terminal New Terminal to open in cmd instread of ps?" or some such.  In my context, your text about the correct feature is exactly what I needed to know.  Thank you for that.  If I saw the bot message without your comment, I would have been very distressed.  There is no reason to re-open this issue.
